builder/bundles.go

```diff
@@ -662,15 +592,15 @@ func (b *Builder) buildBundles(set bundleSet) error {
 emptydir=%s/empty
 imagebase=%s/image/
 outputdir=%s/www/
-`, b.Config.Builder.ServerStateDir, b.Config.Builder.ServerStateDir, b.Config.Builder.ServerStateDir)
-	if cfg.HasServerSection {
```

Comment:
is this ok now because there will always be a section with a default value?

Reply:
Yes, the server section is always present now so this is always true.